### DISCUSSION TITLE
Could not able build theia in alpine image

### DISCUSSION THREAD

**Discussion by dhananjayharel** [eclipse-theia/theia#9769] [Q&A]
### Description
I am trying to build the theia editor using following instructions
https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#quick-start
But when Im trying to do this in alpine container on yarn command i am getting 
the following error

```
$ yarn run clean && yarn run build
$ theiaext clean
$ theiaext build
[compile] error TS2688: Cannot find type definition file for 'linkify-it'.
[compile] theiaext compile exited with code 2
[lint] theiaext lint exited with code 0
```

### Reproduction Steps

**OS and Theia version:**
Alpine-linux 
Nodejs -10
**Diagnostics:**

**Comment by benoitf** [comment]
You may follow examples of docker images using alpine as well:
https://github.com/theia-ide/theia-apps/blob/master/theia-docker/Dockerfile

Theia using native modules you need g++ / python for node-gyp

**Comment by dhananjayharel** [comment]
Thanks,
I already have that modules installed in the container.
I dont have clue why its giving "error  Cannot find type definition file for 'linkify-it'
